tests/status-message/__tests__/main.spec.ts

```diff
@@ -1,6 +1,13 @@
-import { Runner } from "../../../src";
+import { Runner, RunnerFn, Runtime } from "../../../src";
```

Comment:
this and everything below are goofy enough that I think we should create a separate `tests/*` directory called `edge-cases` or something, and move this there, or just write that test in JS instead of TS. But I don't think that's worth doing now.